ShellPkg/Library/UefiShellAcpiViewCommandLib/Parsers/Ras2/Ras2Parser.c

```diff
@@ -114,5 +115,23 @@ ParseAcpiRas2 (
       sizeof (EFI_ACPI_RAS2_PCC_DESCRIPTOR)
       );
     Offset += sizeof (EFI_ACPI_RAS2_PCC_DESCRIPTOR);
+    Count++;

```

Comment:
NIT:
this is also debatable whether we should stop parsing when `Count == *Ras2PccDescriptors`

Reply:
This is a debate. I chose to print all and "Let the user decide". It could be that there is excess garbage data in the table, but count is right. It could be more entries are there and count is wrong. We don't know which one is correct. All we can do is let the user know and decide for themselves. Present all the information to them and leave it to them. Print the excess entries beyond count and if they look like garbage, the user can ignore them. If they look sensible then maybe count is wrong and these extra entries have useful data.

Reply:
Yes right, I still think it would be safer to take the lowest value, but it isn't a major security issue.
I'm letting the comment open for the person who will merge the patch, but lgtm otherwise.

Reply:
yeah - it is up for debate, but as acpiview is pretty much a debug tool 0 the more it shows, the better. if i was an OS it would limit myself to the lower of count or number of entries provided - be conservative. but as this only shows you the info in the table in a readable format, erring on the side of saying more IMHO is better